frontend/packages/component-library/src/button/_BaseButton.tsx

```diff
@@ -90,6 +90,10 @@ export interface ButtonSpecificProps {
   /** (\<button> specific prop)
    *  Button name */
   name?: string;
+  /** (\<button> specific prop)
+   * Forces the button to show the hover state. This is used by the dropdown component to show the
+   * button as active when the dropdown is visible. */
+  forceHover?: boolean;
```

Comment:
Ok, a misunderstanding happened. I though we were talking just about adding active dropdownTrigger button state when dropdown is opened, not editing the Button component itself. We might need to come back to design team and check once more if that's what they really want. (with full context and consequences this will bring)

Reply:
Confirmed with Moshe that 1) we can modify the button if necessary and 2) that we should use the hover instead of active state.